js/src/modal.js

```diff
@@ -484,7 +483,7 @@ class Modal {
 
   _getScrollbarWidth() { // thx d.walsh
     const scrollDiv = document.createElement('div')
-    scrollDiv.className = ClassName.SCROLLBAR_MEASURER
+    scrollDiv.style.cssText = 'position: absolute; top: -9999px; width: 50px; height: 50px; overflow: scroll;'
```

Comment:
Can you remove that change please, as we discussed in the related issue this CSS class is needed 👍